controllers/infra/drivers/stop_instance_test.go

```diff
@@ -31,7 +34,7 @@ func TestDriver_StopInstances(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			d, _ := NewDriver("aws")
+			d, _ := NewDriver("aliyun")
```

Comment:
Seems we need test case for both `aws`+`aliyun` not either?

Reply:
> Seems we need test case for both `aws`+`aliyun` not either?

this test case is useful for both aws and aliyun, only need to change the parameter

Reply:
Run `aws`+`aliyun` both at test file is better, no need to change driver by hand, add one more loop?